index.html

```diff
@@ -249,6 +252,87 @@ <h2>Introduction</h2>
         </li>
       </ul>
 
+      <section class="informative">
+        <h3>Intended Audience</h3>
+
+        <p>
+The intended audience for the Verifiable Credentials Data Model 1.0 W3C Recommendation (Recommendation) 
+is a broad range of professionals interested in furthering
+the application of verifiable credentials technology for use in software apps, agents, and services.
+        </p>
+        <p>
+The primary audience includes software architects, application developers, and user experience (UX)
+specialists;
```

Comment:
I'm not sure that this is actually the primary audience. At least, I'd think we'd be highlighting credential authors and consumers more as opposed to "software architects" and "UX specialists". If I were someone that needed to figure out how to express credential information for some use case or to know how my application will consume it/which parts my application may need to consume (or how to build verification software), this is the spec for me. What do others think?

Reply:
> highlighting credential authors and consumers more as opposed to "software architects" and "UX specialists". 

Who are these people in real life? ...that is, _credential authors_ and _consumers_.  Who are these people on your VC-based app project team? I think we need words (audience roles) that identify with real people and their roles. Further, _credential authors_ and _consumers_ sound like users of a verifiable credential-based system ...not designers and builders of VC-based systems.  The later IMO is who we're targeting. Correct?

> If I were someone that needed to figure out how to express credential information for some use case or to know how my application will consume it/which parts my application may need to consume (or how to build verification software), this is the spec for me. What do others think?

The answer is actually "no" IMO.  The proposed Guide (in addition to the Documentation consisting of the VC data model recommendation, the VC use cases note, and the other 3rd document whose formal name escapes me) is what people (architects and developers) need to read.

Aside: Perhaps _business analysts_ could or should be added to the intended audience statement?

Caution: Think of the intended audience statement as a commitment or contract with the intended readers of the Recommendation. So, for example, if we chose to add any role (e.g. business analyst), we need to make sure there is sufficient content of sufficient relevance and depth to satisfy readers assigned to the particular role.  

For v1.1 specifically, I believe we want to err on the side of a limited set of roles at this point in the review/approval process.

Reply:
> Who are these people in real life? ...that is, credential authors and consumers. Who are these people on your VC-based app project team?

Well, I would not say that they are software architects or UX specialists. Neither of these roles seem to fit to me (at least not as top of mind). A data modeler/data engineer would fit better as a "credential author".

> I think we need words (audience roles) that identify with real people and their roles. Further, credential authors and consumers sound like users of a verifiable credential-based system ...not designers and builders of VC-based systems. The later IMO is who we're targeting. Correct?

Yeah, "consumer" may not be a good name here for the reason you mention. A software engineer, application developer, or security specialist may be interested / participate in writing software to consume or verify VCs in the way that I meant.

My point was that I'd think the primary audience of the spec is those authoring VCs (dealing with data modeling and vocabularies, etc.) and the secondary audience would be those creating software to consume and verify them.

Reply:
> My point was that I'd think the primary audience of the spec is those authoring VCs (dealing with data modeling and vocabularies, etc.) and the secondary audience would be those creating software to consume and verify them.

"authoring VCs" or "designing and building _software systems_ capable of creating, transforming, and verifying VCs?

"authoring VCs" sounds too much like someone who is handcrafting the VC JSON (IMO)

Reply:
> "authoring VCs" sounds too much like someone who is handcrafting the VC JSON (IMO)

That's right -- I think they are the primary audience.

The spec's main focus is on data modeling (it's right there in the title) -- and telling this audience how they need to go about modeling their data so it fits the standard, i.e., how to model the issuer, dates, schemas, status, that the fact that a VC is about one or more credential subjects and where to put that information, about how claims are subject+property+value (and how a value can itself be another subject with an identifier and more claims), showing examples on how to do extensions and include contexts, etc. All of this is about how to "craft" a VC so it will be processable by tools written against the standard.

So, yes, I think the primary audience is those people doing the crafting and the secondary audience is those people doing the processing of what's been crafted.